INSTALL_WINDOWS.md

```diff
@@ -0,0 +1,17 @@
+# Prerequisites on Windows
+
+1. Install [Visual C++ Build Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16): select MSVC Build Tools + Windows 10 SDK
+2. Install [WIX](https://github.com/wixtoolset/wix3/releases/download/wix3112rtm/wix311.exe)
+3. Copy [wixgen.jar](https://github.com/akashche/wixgen/releases/download/1.7/wixgen.jar) to `C:\msi-deps`
+4. Copy all 65 files from `C:\Program Files (x86)\WiX Toolset v3.11\bin` to `C:\msi-deps\wix311-binaries`
+5. Install Cygwin, AdoptOpenJDK, Maven
+
+# Build using Cygwin64 terminal
+
+6. `export JRE=/cygdrive/c/Java/AdoptOpenJDK/jdk-13.0.0.33-hotspot`
```

Comment:
Jdk13/11/8 ?? Not sure what is now most supported JDK for itw. Maybe the  version oculd be wildchar, but I do not insists.

Reply:
it was just an example. The proper way would be to not force users to define a "JRE" variable at all, and either rely on JAVA_HOME, or simply on the PATH. I can remove the version number.

Reply:
If you don't set the variable you get the following error:

```
$ ./build.sh 2.0.0
+ set -e
+ set -o pipefail
+ SCRIPT_SOURCE=./build.sh
+ '[' -h ./build.sh ']'
+ isWindows
++ uname
+ [[ CYGWIN_NT-10.0 == *\N\T* ]]
+ return 0
++++ dirname ./build.sh
+++ cd -P .
+++ pwd
++ cygpath -m /cygdrive/c/git/IcedTea-Web/launchers
+ readonly SCRIPT_DIR=C:/git/IcedTea-Web/launchers
+ SCRIPT_DIR=C:/git/IcedTea-Web/launchers
+ source C:/git/IcedTea-Web/launchers/configure.sh
++ VERSION=2.0.0
++ LOCALIZATIONS='en_US.UTF-8 cs_CZ.UTF-8 pl_PL.UTF-8 de_DE.UTF-8'
++ '[' x2.0.0 == x ']'
++ readonly VERSION=2.0.0
++ VERSION=2.0.0
++ '[' x == x ']'
++ echo 'default jre is necessary'
default jre is necessary
++ exit 1
```

Reply:
@don-vip correctly says that you need to set the JRE variable in order to build. Maybe it can be replaced in this documentation by something like export JRE=*path to JRE on your system*, so that it is not version specific.

Reply:
@don-vip can you take care of that change?

Reply:
What about defaulting to JAVA_HOME if JRE is not defined ?